rollup/rcfg/config.go

```diff
@@ -0,0 +1,33 @@
+package rcfg
+
+import (
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum/common"
+)
+
+// UsingSVM is used to enable or disable functionality necessary for the SVM.
+var UsingSVM bool
```

Comment:
Prefer to use some other name like `UsingScroll`. `SVM` gives others the impression that we are not compatible with `EVM`.